cocotb_build_libs.py

```diff
@@ -218,7 +219,6 @@ def _get_common_lib_ext(include_dir, share_lib_dir):
     libgpilog = Extension(
         os.path.join("cocotb", "libs", "libgpilog"),
         include_dirs=[include_dir],
-        libraries=["cocotbutils"],
```

Comment:
Can you explain why this was here before? Would it have worked fine without it?

Reply:
Yeah, there is no actual dependency there. Not sure why it was there to begin with. Likely we just missed it when we moved to distutils, and it was there before.